netlabel/labels_test.go

```diff
@@ -0,0 +1,69 @@
+package netlabel
+
+import (
+	"testing"
+
+	_ "github.com/docker/libnetwork/testutils"
+)
+
+func TestKeyValue(t *testing.T) {
+	input := []struct {
+		label string
+		key   string
+		value string
+		good  bool
+	}{
+		{"name=joe", "name", "joe", true},
+		{"age=24", "age", "24", true},
+		{"address:1234 First st.", "", "", false},
+		{"girlfriend=", "girlfriend", "", true},
```

Comment:
I know it's just a test, but perhaps use something more generic than "girlfriend" for the name here (and in the other tests)? It's possible people take offend.

Also, following our recommendations for using namespaced/prefixed labels, should we use, or at least include a `com.docker.libnetwork.` or `com.example.` prefix here?